jwst/photom/photom.py

```diff
@@ -187,7 +187,7 @@ def calc_nirspec(self, ftab, area_fname):
 
                         # Populate the photometry keywords
                         self.input.meta.photometry.conversion_megajanskys = \
-                            conv_factor.value
+                            conv_factor
```

Comment:
Is `tabdata['photmjsr']` an astropy quantity?  And if so, does `conv_factor.value` need to be used 2 lines down as well?

Reply:
Doh!  I see it is being _removed_ here.  Nevermind.  Looks good.

Reply:
No, ``tabdata['photmjsr']`` is just a reference to a column in the photom reference file table, which contains a simple scalar value. That instance of ``conv_factor.value`` snuck into the code while I was playing around with adding the use of astropy.units to do the conversion from MJy/sr to uJy/arcsec^2, which I temporarily called ``conv_factor``, until I realised there was already a variable by that name. I obviously just missed this one line when backing out the new use of that variable name.